core/executor.py

```diff
@@ -311,8 +311,9 @@ def RunSimpleCommand(self, cmd_val, cmd_st, run_flags):
         environ = self.mem.GetExported()  # Include temporary variables
 
         if cmd_val.typed_args:
-            e_die('Unexpected typed args passed to external command %r' % arg0,
-                  cmd_val.typed_args.left)
+            e_die(
+                '%r appears to be external. External commands don\'t accept typed args (OILS-ERR-200)'
```

Comment:
Generally speaking I use single quotes unless there's a single quote, then I switch to double quotes, to avoid \

But this is a small thing